packages/sync/src/modules/Term_Relationships.php

```diff
@@ -7,14 +7,32 @@
 
 namespace Automattic\Jetpack\Sync\Modules;
 
-use Automattic\Jetpack\Sync\Defaults;
 use Automattic\Jetpack\Sync\Listener;
 use Automattic\Jetpack\Sync\Settings;
 
 /**
  * Class to handle sync for term relationships.
  */
 class Term_Relationships extends Module {
+
+	/**
+	 * Max terms to return in one single query
+	 *
+	 * @access public
+	 *
+	 * @const int
+	 */
+	const QUERY_LIMIT = 1000;
+
+	/**
+	 * Max mysql integer
+	 *
+	 * @access public
+	 *
+	 * @const int
+	 */
+	const MAX_INT = 999999999;
```

Comment:
@lezama Sorry for the drive-by comment.  Happened to stumble across this PR and noticed this `MAX_INT` definition.  Should this be 2147483647 -- the max value for an unsigned INT in MySQL?  (https://dev.mysql.com/doc/refman/8.0/en/integer-types.html

Reply:
it is a good comment! I don't remember how I decided on that, but I do remember checking that MySQL documentation for it, I think somehow I thought it would be too  magical, I will change it to `2147483647` with a good code comment 👍

Reply:
https://github.com/Automattic/jetpack/pull/14042